### ISSUE TITLE
atob polyfill fails during build on Node 16 with ReferenceError: window is not defined

### ISSUE THREAD

**Issue by zmarty** [braintree/braintree-web#576]
### General information

https://github.com/braintree/braintree-web/blob/main/src/lib/vendor/polyfill.js (commit: cf1b052cc7ddff4ffa62b8da69674ac5d663e0e3)

### Issue description

Node 16 ships with atob and btoa global functions, whereas Node 14 does not.

On our build machine with Node 16 we tried building a [Next.js](https://nextjs.org/) project which uses braintree-web. braintree-web/blob/main/src/lib/vendor/polyfill.js runs as part of the build process, and it fails at this line:

```javascript
var atobNormalized = typeof atob === 'function' ? window.atob : atobPolyfill;
```

The problem here is that in NodeJS 16 atob is defined, and therefore it tries to get it from window.atob, which fails with ```ReferenceError: window is not defined``` because window is not defined during the build process.

```
#11 76.82 > Build error occurred
#11 76.82 ReferenceError: window is not defined
#11 76.82     at Object.<anonymous> (/app/node_modules/braintree-web/lib/vendor/polyfill.js:3:51)
#11 76.82     at Module._compile (node:internal/modules/cjs/loader:1109:14)
#11 76.82     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10)
#11 76.82     at Module.load (node:internal/modules/cjs/loader:989:32)
#11 76.82     at Function.Module._load (node:internal/modules/cjs/loader:829:14)
#11 76.82     at Module.require (node:internal/modules/cjs/loader:1013:19)
#11 76.82     at require (node:internal/modules/cjs/helpers:93:18)
#11 76.82     at Object.<anonymous> (/app/node_modules/braintree-web/lib/create-authorization-data.js:3:12)
#11 76.82     at Module._compile (node:internal/modules/cjs/loader:1109:14)
#11 76.82     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1138:10) {
#11 76.82   type: 'ReferenceError'
```

**Comment by eardenercruk** [comment]
We're also encountering this issue when developing locally with a fresh setup. I've had to downgrade node to 14.16.0 to get it working.

**Comment by jplukarski** [comment]
Thanks for bringing this to our attention. You will have to use Node 14 until we update to Node 16. If you do not want to downgrade your machine you can run `nvm use` prior to the build step to use Node 14. We will leave this issue open until we update (no timeline as of yet).

**Comment by zmarty** [comment]
Yeah we are using Node 14 for now. Thanks!

**Comment by crookedneighbor** [comment]
We'll get this fixed up. Should be a pretty easy fix.

**Comment by crookedneighbor** [comment]
This is fixed in v3.79.0
